sonarqube/action.yml

```diff
@@ -0,0 +1,30 @@
+name: sonarqube
+description: "Run SonarQube Cloud Scan"
+inputs:
+  args:
+    description: "Arguments for SonarQube scan"
+    required: false
+  projectBaseDir:
+    description: "Project base directory"
+    required: false
+  sonarToken:
+    description: "SonarQube token"
+    required: true
+  sonarHostUrl:
+    description: "SonarQube host URL"
+    required: true
+  sonarOrganization:
+    description: "SonarQube organization"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: SonarQube Cloud Scan
+      uses: SonarSource/sonarqube-scan-action@v4.2.1
+      with:
+        args: ${{ inputs.args }}
+        projectBaseDir: ${{ inputs.projectBaseDir }}
+      env:
+        SONAR_TOKEN: ${{ inputs.sonarToken }}
```

Comment:
GitHub composite actions do not have access to the secret so we need to pass them as inputs :(

Reply:
halb so wild